src/sql/workbench/contrib/extensions/electron-browser/extensionsActions.ts

```diff
@@ -0,0 +1,86 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { localize } from 'vs/nls';
+import { IAction, Action } from 'vs/base/common/actions';
+import { ShowViewletAction } from 'vs/workbench/browser/viewlet';
+import { IViewletService } from 'vs/workbench/services/viewlet/browser/viewlet';
+import { IExtension, ExtensionState, IExtensionsWorkbenchService, VIEWLET_ID, IExtensionsViewlet, AutoUpdateConfigurationKey, IExtensionContainer, EXTENSIONS_CONFIG, ExtensionsPolicy, ExtensionsPolicyKey } from 'vs/workbench/contrib/extensions/common/extensions';
+import { IInstantiationService, ServicesAccessor } from 'vs/platform/instantiation/common/instantiation';
+import { IExtensionRecommendation, IExtensionManagementServerService } from 'vs/platform/extensionManagement/common/extensionManagement';
+import { INotificationService, Severity } from 'vs/platform/notification/common/notification';
+import { IOpenerService } from 'vs/platform/opener/common/opener';
+import { CancellationToken } from 'vs/base/common/cancellation';
+import { PagedModel } from 'vs/base/common/paging';
+import { IConfigurationService, ConfigurationTarget } from 'vs/platform/configuration/common/configuration';
+
+export class ShowVisualizerExtensionsAction extends Action {
+
+	static readonly ID = 'workbench.extensions.action.showVisualizerExtensions';
+	static LABEL = localize('showVisualizerExtensions', "Show Visualizers");
+
+	constructor(
+		id: string,
+		label: string,
+		@IViewletService private readonly viewletService: IViewletService
+	) {
+		super(id, label, undefined, true);
+	}
+
+	run(): Promise<void> {
+		return this.viewletService.openViewlet(VIEWLET_ID, true)
+			.then(viewlet => viewlet as IExtensionsViewlet)
+			.then(viewlet => {
+				viewlet.search('@visualizerExtensions');
+				viewlet.focus();
+			});
+	}
+}
+
+export class InstallVisualizerExtensionsAction extends Action {
+
+	static readonly ID = 'workbench.extensions.action.installVisualizerExtensions';
+	static LABEL = localize('installVisualizerExtensions', "Install Visualizer Extensions");
```

Comment:
@kburtram does this look OK to you? I'm happy pending some better suggestion from @alanrenmsft or @yualan and PM, but please confirm.

Reply:
@kevcunnane @yualan @kburtram 

Hi Kevin and all, the language we used here are following the MS writing style guide. According to the guide here are the principles we should follow: 
Our voice hinges on crisp simplicity. Bigger ideas and fewer words. Less head, more heart. 
**Our voice is:**
Warm and relaxed—We’re natural. Less formal, more grounded in real, everyday conversations. Occasionally, we’re fun. (We know when to celebrate.)
Crisp and clear—We’re to the point. We write for scanning first, reading the second. We make it simple above all.
Ready to lend a hand—We show customers we’re on their side. We anticipate their real needs and offer great information at just the right time.

**Example**
Replace this: Invalid ID
With this: You need an ID that looks like this: someone@example.com

Project friendliness
Use contractions: it’s, you’ll, you’re, we’re, let’s. To learn more, see Use contractions.

For all your reference, I have also attached the link to the guide. 
https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28361
https://styleguides.azurewebsites.net/Styleguide/Read?id=2700
https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696

Let me know if you have any other questions or concerns.

Reply:
Yep, the main thing here is being consistent with other extension recommendation language. That's been signed off & shipped in VSCode and Azure Data Studio. Karl had specifically asked to align with that in previous reviews & meetings, so I want this to be verified as being consistent.

Reply:
Updated "Install Visualizer Extensions" to "Install Extensions".